server/sonar-web/src/main/js/app/components/nav/component/ComponentNavBgTaskNotif.tsx

```diff
@@ -0,0 +1,106 @@
+import NavBarNotif from '../../../../components/nav/NavBarNotif';
+import PendingIcon from '../../../../components/icons-components/PendingIcon';
+import { Component } from '../../../types';
+import { STATUSES } from '../../../../apps/background-tasks/constants';
+import { getComponentBackgroundTaskUrl } from '../../../../helpers/urls';
+import { hasMessage, translate, translateWithParameters } from '../../../../helpers/l10n';
+import { Task } from '../../../../api/ce';
+
+interface Props {
+  component: Component;
+  currentTask?: Task;
+  isInProgress?: boolean;
+  isPending?: boolean;
+}
+
+export default class ComponentNavBgTaskNotif extends React.PureComponent<Props> {
+  static contextTypes = {
+    canAdmin: PropTypes.bool.isRequired
+  };
+
+  render() {
+    const { component, currentTask, isInProgress, isPending } = this.props;
+    const canSeeBackgroundTasks =
+      component.configuration != undefined && component.configuration.showBackgroundTasks;
+    const url = getComponentBackgroundTaskUrl(component.key);
+    if (isInProgress) {
+      return (
+        <NavBarNotif className="alert alert-info">
+          <i className="spinner spacer-right text-bottom" />
+          <span
+            dangerouslySetInnerHTML={{
+              __html: canSeeBackgroundTasks
+                ? translateWithParameters('component_navigation.status.in_progress.admin', url)
+                : translate('component_navigation.status.in_progress')
+            }}
+          />
+        </NavBarNotif>
+      );
+    } else if (isPending) {
+      return (
+        <NavBarNotif className="alert alert-info">
+          <PendingIcon className="spacer-right" />
+          <span
+            dangerouslySetInnerHTML={{
+              __html: canSeeBackgroundTasks
+                ? translateWithParameters('component_navigation.status.pending.admin', url)
+                : translate('component_navigation.status.pending')
+            }}
+          />
+        </NavBarNotif>
+      );
+    } else if (currentTask && currentTask.status === STATUSES.FAILED) {
+      if (
+        currentTask.errorType &&
+        currentTask.errorType.includes('LICENSING') &&
+        hasMessage('license.component_navigation.button', currentTask.errorType)
+      ) {
+        return (
+          <NavBarNotif className="alert alert-danger">
+            <span>{currentTask.errorMessage}</span>
+            {this.context.canAdmin && (
+              <Link className="little-spacer-left" to="/admin/extension/license/app">
+                {translate('license.component_navigation.button', currentTask.errorType)}.
+              </Link>
+            )}
+          </NavBarNotif>
+        );
+      } else {
+        return (
+          <NavBarNotif className="alert alert-danger">
+            <span
+              dangerouslySetInnerHTML={{
+                __html: canSeeBackgroundTasks
+                  ? translateWithParameters('component_navigation.status.failed.admin', url)
```

Comment:
blue link in red alert is super ugly

Reply:
Common it's not "super ugly", maybe a "bit ugly", but that's a link... Trust me, it's way better than a blue button! And we have the same colors on issues page.
I don't think we should change the color of links, do you suggest to maybe change a bit the red background ?

Reply:
I'd suggest to use red font for link

Reply:
Ok let me talk to Laura about this

Reply:
Ok let me talk to Laura about this

Reply:
The problem is that red links are for dangerous/irreversible actions.